kumascript/macros/LearnSidebar.ejs

```diff
@@ -168,6 +168,18 @@ var text = mdn.localStringMap({
       'Accessible_multimedia' : 'Accessible multimedia',
       'Mobile_accessibility' : 'Mobile accessibility',
       'Assessment_Accessibility_troubleshooting' : 'Assessment: Accessibility troubleshooting',
+    'Web_performance_making_websites_fast_and_responsive' : 'Web performance — Making websites fast and responsive',
```

Comment:
It would be better IMO to say "performance" rather than "web performance" in all the titles. In this context I think "web" can be assumed.